conda-recipe/bld.bat

```diff
@@ -1,3 +1,7 @@
+@REM new llvm-spirv location
+@REM starting from dpcpp_impl_win-64=2022.0.0=intel_3638 location is env\Library\bin-llvm
+set PATH=%LIBRARY_PREFIX%\bin-llvm;%PATH%
```

Comment:
`%PATH%` is only modified during building. Does `numba_dppy` also need to know where to look for llvm-spirv at runtime?

Reply:
At runtime it uses package `llvm-spirv`.  We decided to not install compiler in runtime. But this package is public. Compiler team does not provide separate package for `llvm-spirv` yet.
Possibly there is inconsistency between llvm-spirv received from `dpcpp` and `llvm-spirv`. @reazulhoque mention some bugs related to this. The second solution could be to install `dpcpp` package in run time.